server/js/sockets.js

```diff
@@ -174,6 +174,13 @@ sockets.init = function socketInit(server) {
       );
     });
 
+    socket.on('send-message', message => {
+      mqttClient.publish(
+        '/v3/camera/primary/message',
```

Comment:
This always publishes to the primary display, regardless of what the user wanted

Reply:
Now that we're going to have all messages show on all displays, the MQTT topic has been changed to just `/v3/camera/message` on the notion V3 MQTT spec page. (this will also need changing on monash-human-power/raspicam#37)